docs/index.rst

```diff
@@ -40,6 +39,16 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
+
+  .. code:: restructuredtext
+
+      .. openapi:: specs/openapi.yml
+          :paths: /persons,/evidence
```

Comment:
Here I have multiple thoughts:

* Could you please rename it into `endpoints` as this word is used everywhere around the text. If we keep `paths`, I'm afraid, it might be a bit confusing with a path to the spec.

* Could you please use 3-space indentation for reStructuredText directive as I use this style in other places in this doc. I.e.:

  ```restructuredtext
  .. code:: restructuredtext

     .. openapi:: specs/openapi.yml
        :endpoints: /persons, /evidence
  ```

* Since there could be multiple endpoints, and it's hard to predict their lengths, I think it might make sense to support per-line list. E.g.:

  ```restructuredtext
  .. code:: restructuredtext

     .. openapi:: specs/openapi.yml
        :endpoints:
           /persons
           /evidence
        :encoding: latin-1
  ```

  In order to do so we need to split `endpoints` content using `re.split(r'\s*\n\s*|\s*,\s*', endpoints)`.

What do you think?